src/Kafka/KafkaInstanceDrawer/CodeSnippets.ts

```diff
@@ -0,0 +1,420 @@
+export const javaConfig = `package org.example;
```

Comment:
@riccardo-forina could you please review the PR, I am treating the code snippet for each client as a const string and displaying that string in the code block, WDYT about this approach, are there any other ways I can try?

Reply:
Yes it's fine